model/accounts_mgmt/v1/registry_credential_resource.model

```diff
@@ -20,4 +20,8 @@ resource RegistryCredential {
 	method Get {
 		out Body RegistryCredential
 	}
+
+	// Delete the registry credentials
```

Comment:
Since this deletes single registry credential, I would change the comment to "Delete the registry credential"

Reply:
Changed